Framework/Crystal/inc/MantidCrystal/IndexPeaksWithSatellites.h

```diff
@@ -0,0 +1,82 @@
+#ifndef MANTID_CRYSTAL_INDEX_PEAKS_WITH_SATELLITES_H_
+#define MANTID_CRYSTAL_INDEX_PEAKS_WITH_SATELLITES_H_
+
+#include "MantidAPI/Algorithm.h"
+#include "MantidDataObjects/Peak.h"
+#include "MantidKernel/System.h"
+#include "MantidKernel/V3D.h"
+
+namespace Mantid {
+namespace Crystal {
+/** IndexPeaksWithSatellites : Algorithm to use the UB saved in the sample
+   associated
+    with the specified PeaksWorkspace, to index the peaks in the workspace.
+
+    @author Shiyun Jin
+    @date   2018-08-17
+
+    Copyright &copy; 2011 ISIS Rutherford Appleton Laboratory &
```

Comment:
This and the other new files should have the new-style copyright notice at the very beginning of the file. They also tell me it is 2019 already. Maybe the years could be updated on the same go?